pkg/noderesourcetopology/cache/overreserve.go

```diff
@@ -97,45 +100,45 @@ func (ov *OverReserve) GetCachedNRTCopy(ctx context.Context, nodeName string, po
 		return nrt, true
 	}
 
-	klog.V(6).InfoS("nrtcache NRT", "logID", klog.KObj(pod), "vanilla", stringify.NodeResourceTopologyResources(nrt))
+	ov.lh.V(6).Info("NRT", "logID", klog.KObj(pod), "vanilla", stringify.NodeResourceTopologyResources(nrt))
```

Comment:
s/vanilla/actual ?

Reply:
will go for "unmodified"

Reply:
decided to change both: "vanilla" -> "fromcache"; "updated" -> "withassumed"